### PR TITLE
Should be off by default | Address the controversial part in read rec…

### PR DESCRIPTION
**Pull request by Shadow243**
I turned off this feature([(was: Cypht: MDN | DSN Missing in compose)](https://avan.tech/item89125)) as a default setting as  asked in 2024-05: https://github.com/cypht-org/cypht/issues/687

### CODE REVIEW

**Comment by marclaporte** [comment]
Once merged to master, please backport to 2.x

File: modules/smtp/modules.php [hunk 1]
```diff
@@ -407,6 +408,7 @@ public function process() {
             $this->out('compose_draft', $draft);
         }
         $this->out('smtp_compose_type', $compose_type);
+        $this->out('user_settings', $settings);
```
Comment (josaphatim) [inline modules/smtp/modules.php:411]: Instead of passing the whole user_settings you can just pass enable_compose_delivery_receipt_setting

Comment (Shadow243) [reply modules/smtp/modules.php:411]: Working on it now

Comment (Shadow243) [reply modules/smtp/modules.php]: @josaphatim done.

**Pull request merged**